CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## Unreleased
 
+## v3.0.0-RC1 (2025-04-14)
+
+### Breaking Changes
+- Remove deprecated triggers and functions support (#1524)
+- Changed default search `DIALECT` to `2` (#1516)
+- Implemented PSR-7 compatible stream abstraction (#1450)
+- Improved pipeline abstractions (#1438)
+- Improved connection handshake (#1431)
+
+### Added
+- Added Redis 7.4 `XREAD` and `HSCAN` interface changes (#1452)
+- Added support for transactions for clustered connections (#1497)
+
+### Fixed
+- Fixed single persistent connection using the same resource (#1512)
```

Comment:
Did you backport this to v2.x?

Reply:
Yes

https://github.com/predis/predis/pull/1515